common-test/src/test/java/io/strimzi/test/StrimziRunnerTest.java

```diff
@@ -63,10 +70,22 @@ public void test0() {
         public void test1() {
             System.out.println("Hello");
         }
+
+        @Namespace("different")
+        @Resources("moreResources")
+        @Test
+        public void test2() {
```

Comment:
Maybe we are really writing our own test framework if our tests need its own tests :-)

Reply:
Well there's non-trivial logic in the `StrimziRunner`, so why not?

Reply:
No problem with that.I just didn't noticed it in the previous reviews.